src/MCPClient/lib/clientScripts/removeUnneededFiles.py

```diff
@@ -2,30 +2,41 @@
 
 import os
 import sys
+import ntpath
```

Comment:
We try to avoid using platform-specific libraries. This can be done using [`os.path`](https://docs.python.org/2/library/os.path.html#os.path.basename)